presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViewsLegacy.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tests.utils.QueryExecutors;
+import org.testng.annotations.Test;
+
+import java.math.BigDecimal;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.HIVE_VIEWS;
+import static io.prestosql.tests.utils.QueryExecutors.onHive;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+
+public class TestHiveViewsLegacy
+        extends BaseTestHiveViews
+{
+    @BeforeTestWithContext
+    public void setup()
+    {
+        // We need to setup sessions for both "presto" and "default" executors in tempto
+        onPresto().executeQuery("SET SESSION hive.legacy_hive_view_translation = true");
+        query("SET SESSION hive.legacy_hive_view_translation = true");
+    }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testShowCreateView()
```

Comment:
Did you mean to inherit these tests from the base class?

Reply:
Those are slightly differnt actually.
On the second though I decided to just have separate test classes and drop common base.